### DISCUSSION TITLE
Recutils as a backend DB?

### DISCUSSION THREAD

**Discussion by philiprhoades** [rwf2/Rocket#2653] [Questions]
[Recutils is a set of tools and libraries to access human-editable, plain text databases called recfiles.](https://www.gnu.org/software/recutils/)

Anyone thought of making use of Recutils as a DB?  I have a small use case for Rocket and I could get started with SQLite but I would REALLY like to use Recutils . .   I don't know Rust but I thought Rocket might be a good way to make something I could actually use while I am (very) slowly getting more familiar with Rust . .

**Comment by SergioBenitez** [comment]
As far as I can tell, there's no library in Rust for using `recfiles`. That would be the first step. Once such a library exists, if it integrates with `deadpool` or exposes a pool of some sort, we'd be happy to add support for it in `rocket_db_pools`. You could, of course, simply add the pool or other such handle to managed state and go from there. In other words, once the base library exists, making use of it with Rocket should be rather straightforward.
